backend/TaskTree/Controllers/ProjectsController.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using TaskTree.Models;
+using TaskTree.Models.Requests;
+using TaskTree.Models.Responses;
+using Task = TaskTree.Models.Task;
+
+namespace TaskTree.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ProjectsController : TaskTreeControllerBase
+    {
+        private readonly TaskTreeContext _context;
+        private readonly IMapper _mapper;
+        private readonly IOptions<AppConfig> _config;
+
+        public ProjectsController(TaskTreeContext context, IMapper mapper, IOptions<AppConfig> config)
+        {
+            _context = context;
+            _mapper = mapper;
+            _config = config;
+        }
+
+        // GET: api/Projects/5
+        [Authorize]
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<ProjectResponse>>> GetProjects()
```

Comment:
Woops, we totally skipped over this endpoint. Delete the commented-out code and replace with something like `var projects = _context.Projects.Where(project => project.UserId == CurrentUserId());` and then change the rest of the logic accordingly. This should return all of a user's projects, so we need to check the Project.UserId field